.dockerignore

```diff
@@ -0,0 +1,18 @@
+node_modules
+*/node_modules
+dist
+*/dist
+manifest-railroad.html
+shell/build
```

Comment:
shell is now shells.  Probably worth reviewing everything in this file to make sure it’s still relevant. Apologies if I busted things moving it wholesale...